### PR TITLE
Tracing

### PR DESCRIPTION
**Pull request by JLerxky**
need https://github.com/cita-cloud/cloud-common-rs/pull/7

### CODE REVIEW

**Review by Jayanring — APPROVED** [review]

**Comment by rink1969** [comment]
更新一下 example/config.toml

**Comment by rink1969** [comment]
update readme

**Review by Jayanring — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ docker build -t citacloud/controller .
 
 ```shell
 $ controller -h
-controller 6.6.0
+controller 6.6.3
```
Comment (rink1969) [inline README.md:14]: update help info

File: example/config.toml [hunk 2]
```diff
@@ -10,6 +11,11 @@ node_address = 'example/node_address'
 storage_port = 50003
 validator_address = 'example/validator_address'
 
+[controller.log_config]
```
Comment (rink1969) [inline example/config.toml:14]: 建议：
```
max_level = "info"
filter = "info"
# rolling_file_path = "./logs"
service_name = "controller"
# agent_endpoint = "127.0.0.1:6831"
```

**Review by rink1969 — APPROVED** [review]

**Pull request merged**